src/components/Release/ReleasePreset.jsx

```diff
@@ -55,16 +55,16 @@ const ReleasePreset = ({
         <Icon icon={icon} />
         <div>
```

Comment:
I know this is merged, it's out of curiosity, is this `<div>` really necessary here?

Reply:
Yes, there's CSS that targets that div to create a flexbox 😊 probably should have labelled it better though.